### DISCUSSION TITLE
7zip or Winrar password access file

### DISCUSSION THREAD

**Discussion by samyueruoribunoki** [community/community#60622] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Hello,

i have a 7zip file, and it needs a password that was revealed,
Is it safe, if i open the file or any 7zip or Winrar file with the required password?

Waiting for your help

Regards

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by GhostOf0days** [comment]
Do you know what the zipped file contains? If not, use a VM to or upload the file [here](https://www.virustotal.com/gui/home/upload) to check for malware.

**Reply by samyueruoribunoki** [reply]
I used the link you sent, and it says only in Malwarebytes analysis that is: Malware.Heuristic.1003,
the other security vendors says that it's Undetected.

Should i ignore or take actions?

**Reply by GhostOf0days** [reply]
Hmm. The only other way to safely check if it's malware besides antivirus tools would be using a VM. It's not a good idea, however, for someone inexperienced. I would just delete the zipped file.

**Reply by samyueruoribunoki** [reply]
Thanks, i will

**Comment by queenofcorgis** [comment]
Hi @samyueruoribunoki ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on zip files. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by GhostOf0days** [reply]
Think it's more so a security issue. I just told OP to delete it. VirusTotal could check for it, but OP said a vendor showed a different result than others. The other way would be to use a VM to perform malware analysis, but it's not a good idea for someone inexperienced to try that.
